primitives/version/src/lib.rs

```diff
@@ -207,6 +235,45 @@ pub struct RuntimeVersion {
 	///
 	/// It need *not* change when a new module is added or when a dispatchable is added.
 	pub transaction_version: u32,
+
+	/// Version of the state implementation used by this runtime.
+	/// Use of an incorrect version is consensus breaking.
+	pub state_version: u32,
+}
+
+/// Struct for decoding without checking `Core` api
+/// version.
+/// This assumes we are using the latest version.
+pub struct RuntimeVersionLatest(pub RuntimeVersion);
+
+impl Decode for RuntimeVersion {
+	fn decode<I: Input>(input: &mut I) -> Result<Self, codec::Error> {
+		let v: RuntimeVersionBasis = Decode::decode(input)?;
```

Comment:
Please decode the members here "manually", like:

`let spec_version = u32::decode(input)?;` bla bla

Reply:
yes can do, I will move the basis struct to the tests where it is used.